examples/grpc/helloworld_client/main.go

```diff
@@ -33,6 +35,10 @@ const (
 )
 
 func main() {
+	go func() {
```

Comment:
Did you prevent client from exiting after response is received? Same for HTTP client.